padcli/command/config.go

```diff
@@ -65,6 +67,11 @@ func RequireConfigFromFileSystem(
 	if !c.HasDefaultFileName() {
 		jetlog.Logger(ctx).Printf("Using config at %s\n\n", c.Path)
 	}
+
+	if *providers.ClusterProvider().GetSelectedClusterName() == "" {
+		providers.ClusterProvider().SetSelectedClusterName(c.Cluster)
```

Comment:
Do we still care about the `--cluster` flag?

Reply:
If we want we can remove, but I didn't want to make that call in this PR. FWIW, I find it useful for testing (e.g. `-c docker-desktop`)